### ISSUE TITLE
ensuring iptables is running fails due to output of `service --status-all`

### ISSUE THREAD

**Issue by pjperger** [chorankates/rouster#84]
On RHEL, the following test will fail:

``` rb
'iptables' => {
  :ensure => 'present',
  :state  => 'started'
}
```

This is because the output of `service --status-all` does not match any of the included regexes for iptables.  Instead, it appears that the iptables rules are simply dumped.  In my example, I included one rule for the nat table and the output of `service --status-all` included the equivalent of `iptables -t nat --list`.

**Comment by chorankates** [comment]
quick testing on a RHEL6.5 system shows these are the potential iptables outputs:

service not started:
```
iptables: Firewall is not running.
```

service started with @pjperger's sample
```
Table: nat
Chain PREROUTING (policy ACCEPT)
num  target     prot opt source               destination         

Chain POSTROUTING (policy ACCEPT)
num  target     prot opt source               destination         

Chain OUTPUT (policy ACCEPT)
num  target     prot opt source               destination         
```

given the 'non-standard' format of output to the `status` command for this service, thinking that a workaround that calls `service <service> status` and looks at exit codes will be the way to fix this issue.

wonder if we'll get better uniformity in exit codes than we did in STD output.

```
[conor@host ~]$ sudo service iptables status                                                                                 
Table: nat
Chain PREROUTING (policy ACCEPT)
num  target     prot opt source               destination         

Chain POSTROUTING (policy ACCEPT)
num  target     prot opt source               destination         

Chain OUTPUT (policy ACCEPT)
num  target     prot opt source               destination         

[conor@host ~]$ echo $?
0
[conor@host ~]$ sudo service iptables stop
iptables: Setting chains to policy ACCEPT: nat             [  OK  ]
iptables: Flushing firewall rules:                         [  OK  ]
iptables: Unloading modules:                               [  OK  ]
[conor@host ~]$ sudo service iptables status
iptables: Firewall is not running.
[conor@host ~]$ echo $?
3
```

**Comment by chorankates** [comment]
only other question is how to determine what services to run `service <service> status` for in the first place.. 

use executable filenames in /etc/init.d/ and /etc/init/ on debian, and /etc/init.d/, [?] on rhel ? ugh

**Comment by chorankates** [comment]
maybe change/overload the way that `Rouster#get_services` works to allow users to specify services they care about, and then using the exit code method on them if they don't get picked up under `--status-all`?

**Comment by chorankates** [comment]
fixed the glitch in #117
